payloads/library/credentials/SMBruteBunny/payload.txt

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+#
+# Title:         SMBruteBunny
+# Author:        Mohamed A. Baset (@SymbianSyMoh)
+# Version:       1.0
+# Targets:       Windows
+# Attack Modes:  HID, RNDIS_ETHERNET
+# Description:   Bruteforces open Windows SMB protocol for valid Credentials against the bruteforce
+#                dictionnary stored in variables "$user_bruteforce_list" and "$pass_bruteforce_list"
+#                then dump the results in text file stored in variable "$password_process_file" then
+#                check for valid or invalid credentials, if valid creds found, store it loot in varaible
+#                "$password_loot_file" then quack it to unlock the machine.
+#
+# LEDS:
+# LED SETUP: Script Setup Stage.
```

Comment:
No need to repeat 'LED' each time. 'SETUP' etc is fine,

Reply:
Done!